invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/components/Document/DocumentStats.js

```diff
@@ -19,59 +16,54 @@ export class DocumentStats extends Component {
 
   fetchStats = async () => {
     const { document } = this.props;
-    const request = {
-      views: {
-        stat: 'record-view',
-        params: {
-          pid_value: document.pid,
-        },
-      },
-    };
+    const pidType = recordToPidType(document);
     try {
-      const response = await http.post(DocumentApis.statsUrl, request);
-      this.setState({ data: response.data });
+      const response = await stats.recordStats(pidType, document.pid);
+      const views = _get(response.data, 'views', {
+        count: '-',
+        unique_count: '-',
+      });
+      this.setState({ views: views });
     } catch (error) {
-      this.setState({ error: error });
+      console.warn(error);
```

Comment:
just to understand: why we don't show the error to the user?

Reply:
I removed the error because it was very weird: after loading the page, suddenly at the bottom you have a yellow box "Something went wrong" and I have no idea what is there or what went wrong!
I think the error should be displayed after an user action, not randomly. In this case it is not an user action, so we should not scare the user. I think it is better to just show "-" instead of the number and fix the issue when happens on our own.
WDYT?

Reply:
true, It also does not make sense to block the whole page because of the error in statistics